package.json

```
@@ -47,6 +52,7 @@
     "@commitlint/config-conventional": "^19.7.1",
     "@svgr/webpack": "^8.1.0",
     "@tanstack/react-query-devtools": "^5.66.0",
```

Biome (1.9.4)

[error] The key @tanstack/react-query-devtools was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)